canvasapi/course.py

```diff
@@ -1312,6 +1312,25 @@ def get_file(self, file, **kwargs):
         )
         return File(self._requester, response.json())
 
+    def get_file_quota(self, **kwargs):
+
+        """
```

Comment:
A tiny nitpick, but [docstrings shouldn't have a blank line between them and the function signature](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings). It looks like it won't interfere with Python's ability to parse it as a docstring, but it's probably worth updating each function to eliminate the blank line for consistency's sake.